src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -258,7 +258,7 @@ SGVector<T> from_gpu(const SGVector<T>& vec)
 			SG_SERROR("Data memory on GPU but no GPU backend registered. \
 						This can happen if the GPU backend was de-activated \
 						after memory has been transferred to GPU.\n");
-			return NULL;
+			return false;
```

Comment:
ok so i wonder how a `(SGVector<T>)false` cast is working out. but yeah we have a problem with the API itself here, because how can we return an `SGVector<T>` if the operation is not successful